ci/environment.yml

```
@@ -2,11 +2,11 @@ channels:
   - conda-forge
 dependencies:
   - conda
-  - python =3.11
+  - python >=3.9
   - pyspark ==3.4.1
   - openjdk ==20.0.2
   - python-duckdb ==0.9.1
-  - polars ==0.19.8
+  - polars >=0.19.3
```

Would this work? (untested)

```suggestion
  - polars >=0.19.3,<=0.19.8  # 0.19.3 on Windows, 0.19.8 on Linux and MacOS
```

---

Nailed it, thanks! https://github.com/coiled/benchmarks/pull/1061/commits/80a60055eb3c989748d6c228580837bd17e19e82